Automattic/jetpack
# Story Block: Pause on long press

Addresses https://github.com/Automattic/jetpack/issues/19949

#### Changes proposed in this Pull Request:
This PR implements a couple of navigation change for mobile users:
- [x] Pause on long press: This allows users to pause the video when pressing the screen with 1 finger, the story starts playing again when the user release their finger
- [x] Allow the next slide button to end the story on the last slide
- [x] Allow Enter to "launch" the story from the embed

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:






* Play a story with and without a video
* Put your finger on the screen to pause the slide, release to play it back
* Try going to the next slide by tapping on the (2/3rd) right side of the screen
* Try going to the previous slide by tapping on the (1/3rd) left side of the screen
* Make sure all the other controls work as expected: play/pause, mute/unmute, exit fullscreen
